mevislab.github.io/content/tutorials.md

```diff
@@ -0,0 +1,225 @@
+| `MDL` | The MeVisLab Definition Language (MDL) is the language for developing basic User Interfaces for `Networks` and `Modules` in MeVisLab. |
+| `Module` | A Module is a single instance providing encapsulated functionalities for a specific purpose. MeVisLab provides thousands of such pre-defined Modules and you can develop your own Modules for extending existing functionalities. Modules provide in- and outputs for connections in a `Network` and/or one or more `Panels` for interacting. |
+| `Panel` | A Panel is a User Interface providing possibilities to interact with MeVisLab. |
+| `Field` | Parameters of Modules are called fields. Several different types of fields are available such as numbers, text, trigger buttons, etc. Public accessible fields can be modified in the `Module Inspector` or Panel of the selected Module.         |
+| `Macro Module` | Macro Modules encapsulate `Networks` including in- and output into a single `Module`. In order to see whats inside a Macro Module, you can open the Context Menu via Right-Click and select {{< menuitem "Show Internal Network" >}}. You can chose to create `Local Macros` and `Global Macros`. |
+| `Local Macro` | Local Macros are only available in your currently opened `Network`. You cannot use the Module Search in MeVisLab to find Local Macros. |
+| `Global Macro` | Global Macros are integrated into MeVisLab and can be used in any future Networks. They are available in Module search. |
+| `Network` | A Network defined at least two connected `Modules`. |
+| `Example Network` | Each `Module` provides an Example Network to see how it can be used. Right-Click on the `Module` and select Help > Show Example Network |
+| `Output Inspector` | The Output Inspector is a quick preview of the output of a specific `Module`. The output can be an image or any other user defined output format. |
+| | For images, a 2D and 3D view including basic interactions is already available. |
+| `Module Inspector` | The Module Inspector shows public available properties of the selected module and their current values. Changes in Module Inspector are applied on the fly. |
+| `Workspace` | The Workspace is the area where you can add and connect `Modules`. Multiple `Networks` are organized in separate Tabs. |
+| `Views Area` | The right side of the `MeVisLab IDE` provides a space to add several predefined panels like `Output-` and `Module inspectors`. |
+| `Debug Output` | The Debug Output shows debugging messages of your `Modules` and the `MeVisLab IDE`. |
+| `Open Inventor` | |
+| `Scene Objects` | |
+| `CSO` | Contour Segmentation Objects (CSOs) |
+| `WEM` | Winged Edge Meshes (WEMs) |
+| `GVR` | Giga Voxel Renderer (GVR) |
+| `Lookup Table (LUT)` | |
+| `Package` | |
+
+
+### MeVisLab IDE User Interface {#tutorial_ide}
+
+First, start MeVisLab IDE. After the Welcome Screen, the standard user interface opens.
+
+![MeVisLab IDE User Interface](/images/IDE1.png "MeVisLab IDE User Interface")
```

Comment:
I would prefer if the images would be grouped into subfolders within the tutorials section. Keeping all at a single location makes sense, if they are used at multiple locations. But I think, that's not the case here.